### PR TITLE
Flutter doc small update

### PR DESCRIPTION
**Pull request by HafizAhsan1**
-- onJourneyResumed() method added in doc

### CODE REVIEW

File: flutter-sdk integration.md [hunk 1]
```diff
@@ -200,22 +203,24 @@ This method takes 4 parameters:
...
-    }
-    
-    func JourneyFinished() {
-        
-    }
+func onJourneyResumed(journeyID: String) {
+  
+}
+
+func onError(error : IDWiseSDKError) {
```
Comment (mheskol) [inline flutter-sdk integration.md:210]: Why do Error and Resume events have "on" prefix while the others below do not?

File: flutter-sdk integration.md [hunk 2]
```diff
@@ -235,15 +239,14 @@ You can initialize the SDK by calling `IDWise.initialize(...)`. It takes "CLIENT
 **startJourney**
 After successfully initializing the SDK with your `CLIENT_KEY` provided by IDWIse, Your app can start an ID verification process by making a call to the `startJourney` method which takes the following parameters:
 
-* **journeyTemplateId** (also called Journey Definition ID): This is a unique identifier that identifies your journey definition. IDWise shares this with you when you register for using IDWise system.
-* **referenceNo**: (Optional) A parameter that you can use to associate an arbitrary identifier (reference number) with the user making the current journey. This is helpful to link the journey back to the user and/or application that started the journey, you will recieve this in the webhook request.
+- **journeyTemplateId** (also called Journey Definition ID): This is a unique identifier that identifies your journey definition. IDWise shares this with you when you register for using IDWise system.
```
Comment (mheskol) [inline flutter-sdk integration.md:242]: We should change the name of this parameter to journeyDefinitionId. Not now as I think this needs code changes but when we have a chance.

File: dynamic-journey-ios.md [hunk 3]
```diff
@@ -156,6 +156,14 @@ After adding our dependency in your Podfile run:
 pod install
 ```
 
+## Modular Approach ( Local Development Pods)
```
Comment (mheskol) [inline dynamic-journey-ios.md:159]: It seems this section is not related to this PR and duplicated from previous one?

File: ios-sdk.md [hunk 4]
```diff
@@ -79,6 +79,14 @@ After adding our dependency in your Podfile run:
 pod install
 ```
 
+## Modular Approach ( Local Development Pods)
```
Comment (mheskol) [inline ios-sdk.md:82]: This section also seems duplicated as in the other PR and doesn't belong here

**Comment by mheskol** [comment]
Where is the Dart code that a hosting app in Flutter needs to write to integrate with us? Can we have a full Dart code snippet showing the parts needed to invoke IDWise SDK?

File: flutter-sdk integration.md (hunk 2 above)
Comment (HafizAhsan1) [reply flutter-sdk integration.md:242]: This section belongs to Android, I'm not sure why they are using journeyTemplateId. @smsibtainrn  @waqar-idwise

**Review by waqar-idwise — APPROVED** [review]
Looks good to me
